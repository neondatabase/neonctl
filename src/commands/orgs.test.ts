import { describe } from 'vitest';

import { test } from '../test_utils/fixtures';

describe('orgs', () => {
  test('list', async ({ testCliCommand }) => {
    await testCliCommand(['orgs', 'list']);
  });
});
