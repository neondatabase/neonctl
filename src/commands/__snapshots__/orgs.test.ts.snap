// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`orgs > list 1`] = `
"- name: Organization 1
  id: 1
- name: Organization 2
  id: 2
- name: Organization 3
  id: 3
"
`;
